modules/node.js

```diff
@@ -67,6 +67,7 @@ Node.prototype.shared = {
 				broadhash: modules.system.getBroadhash(),
 				consensus: modules.peers.getConsensus() || null,
 				height: modules.blocks.lastBlock.get().height,
+				loaded: modules.loader.loaded(),
```

Comment:
`loaded` attribute in a node status is misleading. If the node was loaded properly, so why its accessible. If it would not be loaded, it should not be accessible. I guess you should rename it to `networkLoaded` or some other relevant name.

Reply:
I've taken the same existing name. The behaviour of node module is weird. Using `isLoaded()` method it doesn't work. It should be reviewed.

Reply:
There are multiple things we relates to `loaded` attribute. Is the module is loaded? or blockchain network is loaded? I suggest renaming it to `networkLoaded`

Reply:
Done.